static/resources/js/components/repository/popup-details.directive.js

```diff
@@ -20,31 +20,15 @@
     .module('harbor.repository')
     .directive('popupDetails', popupDetails);
   
-  PopupDetailsController.$inject = ['ListManifestService', '$filter', 'dateLFilter'];
+  PopupDetailsController.$inject = [];
   
-  function PopupDetailsController(ListManifestService, $filter, dateLFilter) {
-    var vm = this;
-    
-    vm.retrieve = retrieve;
-    
-    function retrieve() {
-      ListManifestService(vm.repoName, vm.tag)
-        .success(getManifestSuccess)
-        .error(getManifestFailed);
-    }
-    
-    function getManifestSuccess(data, status) {
-	  console.log('Successful get manifest:' + data);
-	  vm.manifest = angular.fromJson(data.manifest.history[0].v1Compatibility);
```

Comment:
Is this correct? I tried it but it failed, I changed this to simply:

vm.manifest = data.responseJSON;

And then later on refer to its fields like vm.manifest.Id, vm.manifest.OS, and vm.manifest["Docker Version"] etc.